tests/unit/model/tracked-array-test.js

```diff
@@ -190,16 +190,12 @@ module('unit/model/tracked-array', function(hooks) {
     );
 
     chapters.addArrayObserver(this, {
-      willChange: (array, index, removeCount) => {
-        observerWillChange = removeCount;
-      },
       didChange: (array, index, removeCount) => {
         observerDidChange = removeCount;
       },
     });
 
     run(() => chapter2.unloadRecord());
-    assert.equal(observerWillChange, 1, 'removal of the record notified to be removed');
```

Comment:
do we no longer fire willChange?

Reply:
Because we didn't fire it for recordArrays, and did for tracked, so now settling on not firing them